### PR TITLE
docs: Updated FAQ and Query API usage

### PR DESCRIPTION
**Pull request by Anush008**
## Description

As the title goes.

### CODE REVIEW

File: qdrant-landing/content/documentation/examples/qdrant-airflow-astronomer.md [hunk 1]
```diff
@@ -178,12 +178,12 @@ def recommend_book():
...
 
-        result = hook.conn.search(
+        result = hook.conn.query_points(
             collection_name=COLLECTION_NAME,
-            query_vector=preference_embedding,
+            query=preference_embedding,
             limit=1,
             with_payload=True,
-        )
+        ).points
```
Comment (kacperlukawski) [inline qdrant-landing/content/documentation/examples/qdrant-airflow-astronomer.md:186]: Don't we need to specify the name of the vector to use (the `using` parameter)?

File: qdrant-landing/content/documentation/faq/qdrant-fundamentals.md [hunk 2]
```diff
@@ -36,11 +36,8 @@ What Qdrant can do:
 - Apply full-text filters to the vector search (i.e., perform vector search among the records with specific words or phrases)
 - Do prefix search and semantic [search-as-you-type](../../../articles/search-as-you-type/)
 - Sparse vectors, as used in [SPLADE](https://github.com/naver/splade) or similar models
-
-What Qdrant plans to introduce in the future:
-
-- ColBERT and other late-interaction models
-- Fusion of the multiple searches
+- [Multi-vectors](../../concepts/vectors/#multivectors) for ColBERT and other late-interaction models
```
Comment (kacperlukawski) [inline qdrant-landing/content/documentation/faq/qdrant-fundamentals.md:39]:
```suggestion
- [Multi-vectors](../../concepts/vectors/#multivectors), for example ColBERT and other late-interaction models
```

I think we should not make ColBERT the default use case for multivectors. It's applicable if we have multiple embeddings per object.

File: qdrant-landing/content/documentation/faq/qdrant-fundamentals.md [hunk 3]
```diff
@@ -36,11 +36,8 @@ What Qdrant can do:
...
+- Fusion of the [multiple searches](../../concepts/hybrid-queries/)
```
Comment (kacperlukawski) [inline qdrant-landing/content/documentation/faq/qdrant-fundamentals.md:40]:
```suggestion
- Combination of the [multiple searches](../../concepts/hybrid-queries/)
```

Fusion suggests using something like RRF (reranking based on the scores returned by the individual methods), however we can also implement some other techniques.

File: qdrant-landing/content/documentation/examples/databricks.md [hunk 4]
```diff
@@ -81,7 +81,7 @@ def vectorize(partition_data):
         ]
 ```
 
-We're using the [BAAI/bge-small-en-v1.5](https://huggingface.co/BAAI/bge-small-en-v1.5) model for dense embeddings and [prithivida/Splade_PP_en_v1](https://huggingface.co/prithivida/Splade_PP_en_v1) for sparse embeddings.
+We're using the [BAAI/bge-small-en-v1.5](https://huggingface.co/BAAI/bge-small-en-v1.5) model for dense embeddings and [BM42](https://qdrant.tech/articles/bm42/) for sparse embeddings.
```
Comment (kacperlukawski) [inline qdrant-landing/content/documentation/examples/databricks.md:84]: Do we want to use BM42, given the recent discussions around it?

**Review by kacperlukawski — CHANGES_REQUESTED** [review]

File: qdrant-landing/content/documentation/examples/qdrant-airflow-astronomer.md (hunk 1 above)
Comment (Anush008) [reply qdrant-landing/content/documentation/examples/qdrant-airflow-astronomer.md:186]: The collection was created with the default vector.

**Review by kacperlukawski — APPROVED** [review]

**Pull request merged**